Riddle_me_this/settings.py

```diff
@@ -126,6 +126,13 @@
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
 STATIC_URL = '/static/'
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, "static"),
+]
+
+FIXTURE_DIRS = (
+   '/path/to/myapp/fixtures/',
```

Comment:
This path should use "os.path.join.(BASE_DIR, "fixtures"), ".

Reply:
I fixed this , thanks for your respond.